setup.py

```diff
@@ -48,6 +48,6 @@
     ''',
 
     package_data={
-        'hamster_gtk': ('resources/hamster-gtk.gresource',),
+        'hamster_gtk': ['resources/hamster-gtk.gresource'],
```

Comment:
sdist is concatenating some list with the `package_data` items so with tuple it fails on the following error:

```
TypeError: can only concatenate list (not "tuple") to list
```